specs/eip4844/p2p-interface.md

```diff
@@ -188,16 +215,22 @@ Request Content:
 ```
 
 Response Content:
+
+```python
+class BlobSidecars(Container):
+  block_root: Root
+  List[BlobSidecar, MAX_BLOBS_PER_BLOCK]
+
 ```
 (
-  List[BlobsSidecar, MAX_REQUEST_BLOBS_SIDECARS]
+  List[BlobSidecars, MAX_REQUEST_BLOCKS_EIP4844]
 )
 ```
```

Comment:
Render here is broken, triple back quotes not closed see https://github.com/ethereum/consensus-specs/blob/deb82e2f26e9a7cf3062bcce544e842fcfdb72b5/specs/eip4844/p2p-interface.md#blobssidecarsbyrange-v1

Reply:
ffc78e99282e65cc4da92c4dba0743be434d367b